extensions/amp-carousel/0.2/test-e2e/helpers.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const TAG_NAME = 'amp-carousel';
+const SLOTTED_CLASS = 'i-amphtml-carousel-slotted';
+const SPACER_CLASS = 'i-amphtml-carousel-spacer';
+const SCROLLER_SELECTOR = `${TAG_NAME} .i-amphtml-carousel-scroll`;
+
+// TODO(sparhami) Remove once part of the test framework.
```

Comment:
Nice factored out method. I'm still currently working on a layer that knows about AMP concepts like experiments to make this work nicely. I'll ping you in the PR